charts/spire/README.md

```diff
@@ -73,7 +73,14 @@ kubectl delete crds clusterfederatedtrustdomains.spire.spiffe.io clusterspiffeid
 
 ## Upgrade notes
 
-We only support upgrading one major version at a time. Version skipping isn't supported.
+We only support upgrading one major/minor version at a time. Version skipping isn't supported. Please see https://spiffe.io/docs/latest/spire-helm-charts-hardened-about/upgrading/ for details.
+
+### 0.21.X
+
+- If you're doing upgrades as recommended, no special actions should be needed.
+
+- The default value for spire-server.controllerManager.entryIDPrefixCleanup has been changed from "" to false. Upgrades through 0.20.X will have cleaned up old entries in the database and after upgrading to 0.21.X, you may now have
+entries managed manually not get overwritten or cleared out by the spire-controller-manager. If you skipped 0.20.0 right to 0.21.X (unsupported), you will need to manually set the value back to "" to get it to cleanup.
```

Comment:
@kfox1111 Im not sure what this means. Can you elaborate?

Reply:
At 0.21.0+, you can do stuff like:
```shell
kubectl exec -it -n spire-system spire-server-0 spire-server entry create \
  -spiffeID spiffe://example.com/myentry \
  -parentID spiffe://example.com/mynode \
  -selector uid:0
```

And have it not get deleted by the controller manager.

If you didn't follow directions and upgrade through a supported version first, it provides a hint on how to fix it.

Reply:
How about:

- spire-server.controllerManager.entryIDPrefixCleanup's default changed from "" to false.  Upgrades through 0.20.X clean up old entries in the database.  After upgrading to 0.21.X, manual entries will not be overridden by the spire-controller-manager.  Skipping over chart releases (unsupported), requires manual setting of this value to "" to trigger the cleanup.